charts/lakefs/README.md

```diff
@@ -130,5 +150,7 @@ helm upgrade -f my-values.yaml my-lakefs lakefs/lakefs --set kv_upgrade=true
 | `s3Fallback.enabled`                  | If set to true, an [S3Proxy](https://github.com/gaul/s3proxy) container will be started. Requests to lakeFS S3 gateway with a non-existing repository will be forwarded to this container.                                                                                           |                                     |
 | `s3Fallback.aws_access_key`           | An AWS access key to be used by the S3Proxy for authentication                                                                                                                                                                                                                       |                                     |
 | `s3Fallback.aws_secret_key`           | An AWS secret key to be used by the S3Proxy for authentication                                                                                                                                                                                                                       |                                     |
+| `gcpFallback.enabled`                  | If set to true, an [GCP Proxy](https://github.com/GoogleCloudPlatform/cloud-sql-proxy) container will be started.                                                                           |                                     |
+| `gcpFallback.instances`           | The instances to connect to.                                                                                                                                                                                                                       |                                     |
```

Comment:
`instances` means there could be more than a single value right? How do you expect them to be passed?